app/models/user.rb

```diff
@@ -123,5 +128,11 @@ def self.partners_no_new_purchases
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
-
+  private
+  def validate_user_fields
```

Comment:
Assignment Branch Condition size for validate_user_fields is too high. [20.4/15]<br>Inconsistent indentation detected.